src/timesync/timesyncd-server.c

```diff
@@ -60,6 +60,20 @@ ServerAddress* server_address_free(ServerAddress *a) {
         return mfree(a);
 }
 
+static int enable_ntp_server_defer_event(Manager *m, ServerType type) {
+        assert(m);
+        int r;
+
+        r = sd_event_source_set_enabled(m->deferred_ntp_server_event_source, SD_EVENT_ONESHOT);
+                if (r < 0)
+                        log_debug_errno(r, "Failed to reenable system ntp server change event source!");
```

Comment:
this should either return or if it's safe to continue, log as "...event source, ignoring"